composer.json

```diff
@@ -15,15 +15,16 @@
         }
     ],
     "require": {
-        "php": ">=5.6",
+        "php": ">=7.1",
         "markup/addressing": "~0.4.0",
-        "symfony/framework-bundle": "~2.7|~3.0"
+        "symfony/framework-bundle": "~2.7|~3.0|^4"
```

Comment:
I would drop version 2 and add 5, so `^3.4|^4|^5`

Reply:
I don't think this code will actually support Symfony 5